content/10.blackbox.md

```diff
@@ -1 +1,6 @@
-## Tip 8: Do not necessarily consider a DL model as a black box {#blackbox}
+#Rule 8: Do not necessarily consider a DL model as a black box
```

Comment:
This heading reverted to an earlier version.  Please change it back to the `Tip 8` form.